src/Symfony/Component/RateLimiter/CompoundRateLimiterFactory.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\RateLimiter;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class CompoundRateLimiterFactory implements RateLimiterFactoryInterface
+{
+    /**
+     * @param RateLimiterFactoryInterface[] $rateLimiterFactories
```

Comment:
```suggestion
     * @param iterable<RateLimiterFactoryInterface> $rateLimiterFactories
```
Otherwise, it'll enforce `$rateLimiterFactories` to be an `array` instead of `iterable`.